.gitignore

```
@@ -41,7 +41,7 @@ nim_status_client.log
 **/fixtures/*/ipfs
 
 test/ui-test/testSuites/suite_status/config.xml
```

Should it need the same here?
```suggestion
test/ui-test/testSuites/*/config.xml
```

---

Yeah, but it's not necessary to amend this commit further. It can be changed in one of my other PRs